### PR TITLE
[Chore] Fix code blocks, pt 1

### PR DESCRIPTION
**Pull request by kodster28**
### Summary

Large block, mostly CF1

### CODE REVIEW

File: src/content/docs/ai-gateway/tutorials/deploy-aig-worker.mdx [hunk 1]
```diff
@@ -81,8 +86,9 @@ import OpenAI from "openai";
 export default {
 	async fetch(request, env, ctx) {
 		const openai = new OpenAI({
-		  apiKey: env.OPENAI_API_KEY,
-		  baseURL: "https://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_id}/openai" // paste your AI Gateway endpoint here
+			apiKey: env.OPENAI_API_KEY,
+			baseURL:
+				"https://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_id}/openai", // paste your AI Gateway endpoint here
```
Comment (irvinebroque) [inline src/content/docs/ai-gateway/tutorials/deploy-aig-worker.mdx:91]: should this be on same line?

**Review by irvinebroque — APPROVED** [review]
nice one

File: src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx [hunk 2]
```diff
@@ -108,9 +110,11 @@ resolver #2
 
 On Windows, open a PowerShell window and run `ipconfig`. The DNS servers should be set to WARP's local DNS proxy IPs.
 
-```powershell null {17-18}
+```powershell
 PS C:\> ipconfig
+```
 
+```powershell {15-16} output
```
Comment (ranbel) [inline src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx:117]: should this be 16-17? (Not sure how the new highlighting component works. The other sh blocks use "n-1" logic.)

File: src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx [hunk 3]
```diff
@@ -189,8 +199,11 @@ utun3: flags=8051<UP,POINTOPOINT,RUNNING,MULTICAST> mtu 1280
 
 On Windows, run `ipconfig`. When WARP is turned on, you will see an adapter called `CloudflareWARP` with IP address `172.16.0.2`.
 
-```powershell null {14}
+```powershell
 PS C:\> ipconfig
+```
+
+```powershell {12} output
```
Comment (ranbel) [inline src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx:206]: same question regarding the line highlighting

**Review by ranbel — APPROVED** [review]

File: src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx (hunk 3 above)
Comment (kodster28) [reply src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx:206]: Let me check.

File: src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx (hunk 2 above)
Comment (kodster28) [reply src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx:117]: Let me check

File: src/content/docs/ai-gateway/tutorials/deploy-aig-worker.mdx (hunk 1 above)
Comment (kodster28) [reply src/content/docs/ai-gateway/tutorials/deploy-aig-worker.mdx:91]: Likely! Will investigate

Comment (kodster28) [reply src/content/docs/ai-gateway/tutorials/deploy-aig-worker.mdx:91]: Ah... it's a setting with our new built-in formatting that just affects the readability.

File: src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx (hunk 3 above)
Comment (kodster28) [reply src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx:206]: confirmed, looks good

File: src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx (hunk 2 above)
Comment (kodster28) [reply src/content/docs/cloudflare-one/connections/connect-devices/warp/configure-warp/route-traffic/warp-architecture.mdx:117]: confirmed, looks good

**Comment by kodster28** [comment]
relates to #16210

**Pull request merged**